src/components/Settings/Delete/index.tsx

```diff
@@ -51,6 +54,7 @@ const CREATE_BURN_PROFILE_TYPED_DATA_MUTATION = gql`
 `;
 
 const DeleteSettings: FC = () => {
+  const [showWarningModal, setShowWarningModal] = useState(false);
```

Comment:
```suggestion
  const [showWarningModal, setShowWarningModal] = useState<boolean>(false);
```

Reply:
Thanks for comment!
I will count for future requests your preferred type annotation approach!


In some projects and by inner feeling it is better to avoid explicit typings if there is already 100% implicit type detected

TS/eslint even has error: Avoid explicit typing for implicitly derived
Something like this**

That is why I already prefer false, instead of <boolean>(null)

For simple types - it is easy to do
But if your function has complex Pick, Omit, Params, etc. - we usually believe IDE and don't annotate))

And don't worry - I use explicit typings for things like lodash, or Json transformations if "they"(IDE and TS itself) cannot derive type

🔥🤓

Reply:
@bigint found, it was "inferable", not "derived"

prefer "no inferrable types" rule - 🔥 

https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-inferrable-types.md

Reply:
Oh this makes sense @andriishupta, we can make a separate PR to fix this 🚀

Reply:
no need for separate PR - there are thousands of rules 🥵
just if you see sense it in - I will omit such things in "obvious" inferred places in my next PRs

thanks for checking this out 🤓